DTS-STN/Service-Canada-Labs
# update splash image

# Description

Updated the splash page background image. The old image has a poor quality, swapping it with a better quality image.

For some reason, cypress is complaning about the VA overview page missing a level-one heading. However the page does have a h1 title, not sure why it's throwing the error. I commented out the test for the VA page for now, will investigate this issue in another pr. 

## Test Instructions

1. yarn dev
2. visit the index page
3. the style of the image didn't change, but the image should have better quality
